Data/GraphViz/Attributes/Arrows.hs

```diff
@@ -16,6 +16,7 @@ import Data.GraphViz.Internal.Util (bool)
 import Data.GraphViz.Parsing
 import Data.GraphViz.Printing
 
+import Prelude hiding ((<>))
```

Comment:
This fails for GHC 7.4.2 (and results in a warning for 7.6.3).

I can think of a few solutions for this:

1) No longer support 7.4.* and live with the warning for 7.6.* up until 8.4.1
2) Depend upon `base-compat` (which is a dependency of `wl-pprint-text` already), but even then we'd need to do `#if` CPP macros everywhere until base 4.9
3) Use `base-compat-batteries` (which drags in more dependencies)
4) Solve ivan-m/wl-pprint-text#21

I think 1) would be an adequate solution for now with 4) as the long-term solution.

Reply:
Dropped 7.4.* as per option 1